### PR TITLE
Adding a deploy dashboard CTA

### PR DESCRIPTION
**Pull request by AdityaHegde**
- [x] Add a basic modal to tell the user to use CLI to deploy
- [x] Align with the mocks

### CODE REVIEW

File: web-common/src/features/dashboards/workspace/DeployDashboardOverlay.svelte [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  function close() {
+    projectShareStore.set(false);
+  }
+</script>
+
+<Dialog on:cancel={close} size="md">
+  <div class="" slot="body">
+    <div class="text-center text-2xl p-6">
+      Please use the CLI to deploy your project by running `rill deploy`
```
Comment (djbarnwal) [inline web-common/src/features/dashboards/workspace/DeployDashboardOverlay.svelte:13]: Can we improve the copy and design of this part? Current dialog text looks a bit too big 

image

Can we use the `CLICommandDisplay` component to show `rill deploy` with a copyable icon?

Also let's add a button which redirects the user to Rill Deploy docs.

**Review by djbarnwal — CHANGES_REQUESTED** [review]
Have suggested some design changes for the deploy button.

File: web-common/src/features/dashboards/dashboard-stores.ts [hunk 2]
```diff
@@ -302,3 +302,5 @@ export function useDashboardStore(
 }
 
 export const calendlyModalStore: Writable<string> = writable("");
```
Comment (djbarnwal) [inline web-common/src/features/dashboards/dashboard-stores.ts:304]: Also do we still need Calendly components and store now in the codebase? If not, we can remove those as well.

**Review by djbarnwal — CHANGES_REQUESTED** [review]

File: web-common/src/features/dashboards/workspace/DeployDashboardOverlay.svelte (hunk 1 above)
Comment (AdityaHegde) [reply web-common/src/features/dashboards/workspace/DeployDashboardOverlay.svelte:13]: The design docs have these pretty big. We can check quickly with janet and figure this out.

**Review by djbarnwal — APPROVED** [review]
Merging this. We can circle back on the deploy modal content design later.

**Pull request merged**